modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -121,40 +125,64 @@ image::manage-security/configureLDAPencryptionPullDownMenu.png[,200,align=left]
-(See xref:learn:security/certificates.adoc[Certificates] for information.)
-If *Paste Cert* is selected, the panel expands vertically, to reveal the *Certificate Text* field:
+If *Couchbase* is selected, the root certificate for the Couchbase-Server cluster is used for validation: this assumes that the same root certificate is installed for the LDAP server.
+(See xref:learn:security/certificates.adoc[Certificates] for information on root certificates.)
++
+*Paste Cert* should be selected if the LDAP server root certificate is _not_ the same certificate as has been installed for the Couchbase-Server cluster.
+When *Paste Cert* is selected, the panel expands vertically, to reveal the *LDAP Server Root Certificate* field:
 +
 [#certificate-text-field]
 image::manage-security/certificateTextField.png[,400,align=left]
 +
-The text of the appropriate certificate should be copied and pasted, in PEM format, into the *Certificate Text* field.
+The text of the LDAP server root certificate should now be copied and pasted, in PEM format, into the *LDAP Server Root Certificate* field.
+
+* *Contact LDAP host*.
+The following options are provided, to determine how Couchbase Server will contact the LDAP host.
 
-* *Contact LDAP host anonymously*.
+** *Anonymously*.
 Checking this checkbox causes Couchbase Server to attempt to contact the LDAP host anonymously.
 However, the attempt succeeds only if supported by the LDAP server.
 
-* *LDAP DN*.
-The LDAP _Distinguished Name_ of the user to perform user-search and groups synchronization.
-This user needs to have _read only_ access to the LDAP server, in order to be able to search for users and groups.
-
-* *Password*.
-The password for the user who has been specified in the *LDAP DN* field.
-
+** *Credentials*.
+Checking this checkbox allows credentials to be specified by the user, and submitted to the LDAP host for authentication.
+The UI expands, to display two additional fields:
++
+image::manage-security/contactLDAPhostCredentials.png[,450,align=left]
++
+*Bind DN* requires the LDAP _Distinguished Name_ of the user who will perform user-search and groups synchronization.
+This user needs to have read only access to the LDAP server, in order to be able to search for users and groups.
+The `uid` will be the username with which the user has logged into Couchbase Server.
++
+*Password* requires input of the user's password, as stored on the LDAP server.
++
 With data entered into these fields, the dialog might appear as follows:
-
++
 [#configure-ldap-dialog-half-complete]
 image::manage-security/configureLDAPhalfComplete.png[,520,align=left]
++
+[#client-certificate]
+** *Client Certificate* allows Couchbase Server to authenticate with LDAP by means of a client certificate.
+For information on client-certificate configuration, see xref:manage:manage-security/configure-server-certificates.adoc[Configure Client Certificates].
```

Comment:
I'm not sure it is the right place for this link here. Client-cert authentication for Couchbase users is a completely different topic and this link rather confuses than helps.

Reply:
I've removed.